src/App.js

```diff
@@ -1,25 +1,64 @@
-import logo from './logo.svg';
 import './App.css';
+import contactsData from './contacts.json';
+import { useState } from "react";
 
 function App() {
+
+  const [contactsToDisplay, setContactsToDisplay] = useState(contactsData)
+
+  let fiveContacts = contactsData.slice(0,5)
+  
+
+  const deleteContact = (contactId) => {
+   
+    // contactsToDisplay.push(); // NEVER MODIFY STATE DIRECTLY !
+
+    const newList = contactsToDisplay.filter( (element) => {
+        return element.id !== contactId;
+    });
+
+    setContactsToDisplay(newList);
+  }
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+
+    {fiveContacts.map((contactsObj) => {
```

Comment:
Geralmente queremos mapear um `state` pois isso fara com que a view seja dinâmica, assim quando você deletar um contato a view atualiza

```suggestion
    {contactsToDisplay.map((contactsObj) => {
```